<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWG</class>
 <widget class="QWidget" name="SearchWG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="search_range_groupBox">
     <property name="title">
      <string>Search Range</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="match_control_groupBox">
     <property name="title">
      <string>Match Control</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="case_sensitive_cbx">
        <property name="text">
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="match_whole_word_cbx">
        <property name="text">
         <string>Match Whole words</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="use_regular_express_cbx">
        <property name="text">
         <string>Use Regular Expression</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="file_groupBox">
     <property name="title">
      <string>Files</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
      <item>
       <widget class="QLineEdit" name="file_filter_le"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>341</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="time_groupBox">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDateTimeEdit" name="start_time_dte">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>2</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss zzz</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;&lt; - &gt;&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="end_time_dte">
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>2</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>yyyy-MM-dd HH:mm:ss zzz</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>358</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="operation_groupBox">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QPushButton" name="search_btn">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>484</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="search_le"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
